src/blocks/reader-registration/index.php

```diff
@@ -184,9 +184,7 @@ function render_block( $attrs, $content ) {
 		<?php if ( $registered ) : ?>
 			<div class="newspack-ui__box newspack-ui__box--success newspack-ui__box--text-center">
 				<span class="newspack-ui__icon newspack-ui__icon--success">
-					<svg aria-hidden="true" width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-						<path d="M16.7 7.1l-6.3 8.5-3.3-2.5-.9 1.2 4.5 3.4L17.9 8z" />
-					</svg>
+					<?php \Newspack\Newspack_UI_Icons::print_svg( 'check' ); ?>
```

Comment:
Since this class is used several times throughout this file, consider adding a `use Newspack\Newspack_UI_Icons;` statement at the top so you can eliminate the prefix.